src/flamenco/runtime/context/fd_exec_epoch_ctx.c

```diff
@@ -68,11 +68,11 @@ fd_exec_epoch_ctx_new( void * mem,
   self->layout = *layout;
 
   fd_features_disable_all( &self->features );
-  fd_features_enable_hardcoded( &self->features );
+  fd_features_enable_hardcoded( &self->features, self->epoch_bank.cluster_version );
 
   fd_epoch_bank_new( &self->epoch_bank );
 
-  self->epoch_bank.cluster_version = 2000;
+  self->epoch_bank.cluster_version = 1900;
 
```

Comment:
Same here.. we would probably roll this forward to the latest version.. so that by default things work with newer ledgers without arguments